0.data-download/README.md

```diff
@@ -0,0 +1,30 @@
+### Description of the Data ###
+
+# DepMap Public 22Q4
+
+This DepMap release contains data from in vitro studies of genetic dependencies in cancer cell lines using CRISPR/Cas9 loss-of-function screens from project Achilles, as well as genomic characterization data from the CCLE project.
+
+Data resource:
+https://depmap.org/portal/download/all/
+
+
+
+
+### Files used in the WayScience Project ###
+
+### CRISPR_gene_dependency.csv ###
+
+This data comprises scores from individual CRISPR gene knockout screens in cancer cell lines for every gene across many different ages and cancer types. The values represent gene dependency probability estimates for cell survival and growth for all models in the integrated gene effect. The higher the score, the greater the gene dependency. 
```

Comment:
```suggestion
This data comprises scores from individual CRISPR gene knockout screens in cancer cell lines for every gene across many different ages and cancer types.
The values represent gene dependency probability estimates for cell survival and growth for all models in the integrated gene effect.
The higher the score, the greater the gene dependency. 
```

We follow the [one-sentence-per-line convention](https://news.ycombinator.com/item?id=31808093) in markdown files.